pkg/ansible/roles/mongodb/defaults/main.yml

```diff
@@ -0,0 +1,34 @@
+---
+mongodb_version: "3.4"
+mongodb_package: "mongodb-org"
+apt_key_fingerprint: "0C49F3730359A14518585931BC711F9BA15703C6"
+apt_keyserver: "keyserver.ubuntu.com"
+distribution_name: "{{ansible_distribution|lower }}"
+distribution_codename: "{{ ansible_distribution_release|lower }}"
+distribution_major: "{{ ansible_distribution_major_version }}"
+server_arch: "amd64,arm64"
+
+# MongoDB Repos
+mongodb_apt_repo: "deb [arch={{ server_arch }}] http://repo.mongodb.org/apt/{{ distribution_name }} {{ distribution_codename }}/{{ mongodb_package }}/{{ mongodb_version }} {{'main' if ansible_distribution == 'debian' else 'multiverse'}}"
+mongodb_yum_base_url: "https://repo.mongodb.org/yum/{{ ansible_os_family|lower }}/$releasever/{{ mongodb_package }}/{{ mongodb_version }}/{{ ansible_architecture }}"
+mongodb_dnf_base_url: "https://repo.mongodb.org/yum/{{ ansible_os_family|lower }}/7/{{ mongodb_package }}/{{ mongodb_version }}/{{ ansible_architecture }}"
+
+# MongoDB running config
+mongodb_storage_path: /data/db
```

Comment:
This has been changed to `/data/db/main` in the container recently.

Reply:
Done.